### PR TITLE
feature:sem-073

### PR DESCRIPTION
**Pull request by davidcmbot11**
Description
We need to implement a component that contains the elements shown in the image, thus containing all the form field
#73

### CODE REVIEW

**Comment by davidcmbot11** [comment]
this is my storybook 
https://ditmar.github.io/backoffice-openscience/branch-sem-073/?path=/story/ui-components-molecules-bibliographyform--basic

File: src/ui/components/molecules/BibliographyForm/bibliograpyform.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import InputField from '../../atoms/InputField/InputField';
+import type { IFormProps } from './types/Iprops';
+import styles from './bibliograpyform.module.scss';
+
+function Form({ fields }: IFormProps) {
+  return (
+    <form className={styles.form__field}>
+      {fields.map((field) => (
```
Comment (Ditmar) [inline src/ui/components/molecules/BibliographyForm/bibliograpyform.tsx:8]: i am not sure about this approach, please change this form is unique

**Review by Ditmar — CHANGES_REQUESTED** [review]
Don´t use a for to create the fields,  this form is unique only create the HTML completely and check the way to catch all data, for after fetching data

**Comment by davidcmbot11** [comment]
new link 
https://ditmar.github.io/backoffice-openscience/branch-sem-073/?path=/story/ui-components-molecules-bibliographyform--bibliography

**Review by Ditmar — APPROVED** [review]

**Pull request merged**